profiles/standard/standard.install

```diff
@@ -193,7 +193,7 @@ function standard_install() {
 
   // Create an image field named "Image", enabled for the 'article' content type.
   // Many of the following values will be defaulted, they're included here as an illustrative examples.
-  // See http://api.drupal.org/api/function/field_create_field/7
+  // See https://api.backdropcms.org/api/backdrop/core%21modules%21field%21field.crud.inc/function/field_create_field/1
```

Comment:
api.backdropcms.org makes strange URLs (`/core%21modules%21field%21field.crud.inc` etc). Maybe that should be fixed before adding this to docs.